src/components/content/heroes/aurora/aurora-components.tsx

```diff
@@ -63,27 +62,5 @@ export const flaresByWeather: { [key in WeatherType] } = {
   [WeatherType.Cloudy]: <Clouds amount={20} />,
   [WeatherType.Snowy]: <Snow amount={50} />,
   [WeatherType.Rainy]: <Rain amount={550} />,
-  [WeatherType.NotSet]: (
-    <>
-      <Flare
-        stepSize={20}
-        flareType={FlareType.DARK}
-        x={'50vw'}
-        y={'50vw'}
-        size={150}
-        rotation={30}
-        animationOffset={3}
-      />
-      <Flare
-        stepSize={0}
-        flareType={FlareType.DARK}
-        x={'70vw'}
-        y={'300px'}
-        size={100}
-        rotation={80}
-        animationOffset={14}
-      />
-      <DefaultFlares />
-    </>
-  ),
+  [WeatherType.NotSet]: <DarkDefaultFlares />,
```

Comment:
I am not sure if this improves the encoupling of aurora and the easteregg. since they are stll coupled in aurora-components but we hide it by putting the component into aurora. I would rather not do this.